src/layouts/about.jsx

```diff
@@ -2,7 +2,186 @@ import React from 'react';
 
 const About = () => (
   <div className="about">
-    <h2>About us!</h2>
+    <div className="group-photo-container text-center">
+      <img className="img-responsive group-photo" src="/images/about-cover.png" alt="A group discussion about Signsfive" />
+    </div>
+
+
+    <div className="container">
+      <section className="container py-5 px-5">
+        <h3 className="tracked pb-2">About SignsFive</h3>
+        <p>
+          <strong>
+            Make the search and share for STE(A)M (Science, Technology,
+            Engineering, <em>Art</em>, Math) signs easier.
+          </strong>
+        </p>
+        <p>
+          In fields where new technologies and technical terms are
+        always emerging, SignsFive will be a place where d/Deaf/HoH,
+        interpreters and ASL students (and hearing users of ASL) can
+        find signs for STE(A)M terms and contribute to the growing collection.
+        </p>
+      </section>
+      <div className="row py-5">
+        {/* <!--about-our-story--> */}
+        <div className="col-md-6 px-5">
+          <h3 className="tracked pb-2">Our Story</h3>
+          <p>
+            <strong>In early 2017, a group of deaf/hard-of-hearing/deafblind and hearing professionals began meeting weekly at <a href="https://chihacknight.org/">ChiHackNight</a> to discuss what issues that haven’t been addressed well.</strong>
+          </p>
+
+          <p>
+            As a developer/engineer, Jason Salerno shared that he had been frustrated that
+            the recurring problems for existing online/web sign language dictionaries for STE(A)M:
+          </p>
+
+          <ol>
+            <li>
+              Not active anymore because there are no more grants to
+                  support or admins have moved on
+            </li>
+            <li>Not user-friendly nor mobile-friendly
+
+            </li>
+            <li>Not fast enough to adds signs, especially in STE(A)M field where
+                it’s ever changing
+            </li>
+          </ol>
+          <p>
+            Initially called OpenSigns, the group hatched SignsFive, a community-sourced
+            open to anyone in the sign language community to contribute, share and discuss
+            STE(A)M signs.
+          </p>
+
+        </div>
+
+
+        {/* <!--about-our-name--> */}
+        <div className="col-md-6">
+          <h3 className="tracked pb-2">Our Name</h3>
+          <p>
+            <strong>
+              You might be wondering, {'"'}What’s the story behind the name?{'"'}
```

Comment:
its bit confusing reading this from code wise - can we simplify this something like 
```
 <strong>
   {`You might be wondering, "What’s the story behind the name?"`}
</strong>
```
or
```
 <strong>
   You might be wondering, {`"What’s the story behind the name?"`}
</strong>
```